webpack.config.production.js

```diff
@@ -21,6 +21,11 @@ module.exports = (env) => {
                     from: 'vendor/**/*.*',
                     to: 'js/',
                 },
+                {
+                    // placeholders substituted during deployment
+                    from: 'bundles.template.json',
+                    to: 'bundles.template.json',
```

Comment:
Should the `to` file contain `.template` in the name or be just `bundles.js`?

Reply:
yes, then in the deploy plan we pick up bundles.template.json, replace the placeholders and rename the file without the "template" part.

Reply:
Ok :+1:

Reply:
This copy just makes sure it gets to the "runtime_build" directory.